server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/CICSVisitor.java

```diff
@@ -341,7 +342,7 @@ private void areaBWarning(@NonNull List<Token> tokenList) {
                         throwException(
                             token.getText(),
                             locality,
-                            messageService.getMessage("CobolVisitor.AreaBWarningMsg"))));
+                            MessageTemplate.of("CobolVisitor.AreaBWarningMsg"))));
```

Comment:
Similarly here `throwException(locality, MessageTemplate.of("CobolVisitor.AreaBWarningMsg", token.getText()))`?